src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -235,13 +241,16 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
       throw new Error(error.message);
     }
   }, [
+    setAuthCredentials,
+    campaignFeatures?.campaignName,
```

Comment:
hmm, just curious, do we need this?

Reply:
Yep, this was automatically added because of the check on the campaign name in the effect

Reply:
oooh, okok! i see!